rest/changes_api_test.go

```diff
@@ -3213,13 +3213,9 @@ func TestChangesActiveOnlyWithLimitLowRevCache(t *testing.T) {
 	response = rt.SendAdminRequest("PUT", "/db/activeDoc5", `{"channel":["PBS"]}`)
 	assertStatus(t, response, 201)
 
```

Comment:
It looks like we need another WaitForChanges on line 3187 for this test.

Reply:
Makese sense, added changes to wait for changes.